hands-on/gemm/gemmNN/gemmNN.cpp

```diff
@@ -0,0 +1,116 @@
+#define N 1000
```

Comment:
Please create 
0-gemmNT-omp-serial
1-gemmNT-omp-thread
2-gemmNT-omp-target
to archive different versions.

Reply:
Do you mean gemmNN-omp-serial, thread, and target respectively?

Reply:
yes. gemmNN